crates/wiggle/macro/src/lib.rs

```diff
@@ -168,7 +168,30 @@ pub fn from_witx(args: TokenStream) -> TokenStream {
         quote!()
     };
 
-    TokenStream::from(quote! { #code #metadata })
+    let mut ret = quote! { #code #metadata };
+
+    if std::env::var("WIGGLE_DEBUG_BINDGEN").is_ok() {
```

Comment:
Wish I had done this years ago

Reply:
I should wrap this up in a helper as well since I'm starting to litter this all over the macros I work on...